tests/10-vcdm2.js

```diff
@@ -147,29 +141,38 @@ describe('Verifiable Credentials Data Model v2.0', function() {
     });
 
     describe(name, function() {
-      it.skip('Conforming document (compliance): VCDM "MUST be enforced." ("all relevant normative statements in Sections 4. Basic Concepts, 5. Advanced Concepts, and 6. Syntaxes")', async function() {
+      it.skip('Conforming document (compliance): VCDM "MUST be enforced." ' +
+        '("all relevant normative statements in Sections 4. Basic Concepts, ' +
+        '5. Advanced Concepts, and 6. Syntaxes")', async function() {
       });
       // Basic
-      it.skip('"A serialization format for the conforming document MUST be deterministic, bi-directional, and lossless as described in Section 6. Syntaxes."', async function() {
+      it.skip('"A serialization format for the conforming document MUST be ' +
+        'deterministic, bi-directional, and lossless as described in Section' +
+        ' 6. Syntaxes."', async function() {
```

Comment:
```suggestion
      it.skip('A serialization format for the conforming document MUST be ' +
        'deterministic, bi-directional, and lossless as described in Section ' +
        '6. Syntaxes.', async function() {
```

Reply:
this is basically everywhere we might want to just do all of this one commit and not via github comments.